src/java.desktop/share/classes/sun/print/RasterPrinterJob.java

```diff
@@ -1164,6 +1170,11 @@ protected boolean isSupportedValue(Attribute attrval,
                                           attrset));
     }
 
+    protected boolean isAttributeCategorySupported(Class<? extends Attribute> category) {
```

Comment:
hmm. Not sure why this isn't private, and why it has the same name as the PrintService method which can be confusing.
It has one purpose and one use which is to ask if Chromaticity is supported so it can be clearer.